resources/ext/build-scripts/install-dropsonde.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+FORGE_API="forgeapi-cdn.puppet.com"
```

Comment:
I think this should just be `forgeapi.puppet.com`? Not sure where the `-cdn` bit comes from.

Reply:
All things related to the Forge API I take from here https://forgeapi.puppet.com/#section/Features.
`forgeapi-cdn.puppet.com` supports the IPv6 domain.